### DISCUSSION TITLE
mod translation

### DISCUSSION THREAD

**Discussion by ChaosPrometheus** [Aizistral-Studios/Enigmatic-Legacy#216] [Show and tell]
Hi, I have a translation of the mod into Russian
[en_us.txt](https://github.com/Aizistral-Studios/Enigmatic-Legacy/files/7472484/en_us.txt)
after downloading the file from txt, rename it to json

**Comment by Aizistral** [comment]
Блин, братан, это немного не так делается. Форкни репозиторий и сделай нормальный Pull Request. Тебе буквально просто нужно назвать файл ru_ru.json и залить его вот сюда: https://github.com/Aizistral-Studios/Enigmatic-Legacy/tree/1.16.X/src/main/resources/assets/enigmaticlegacy/lang в своём форке. И да, это важно - как минимум мне будет проще показать места где по тем или иным причинам нужны правки.

**Comment by ChaosPrometheus** [comment]
привет я не могу сделать,я в гидхабе нуб

чт, 4 нояб. 2021 г. в 20:59, Aizistral ***@***.***>:

**Reply by Aizistral** [reply]
Господи, тут даже нуб разберётся. Мог бы сам загуглить, но если даже это так тяжело - вот тебе видео которое вроде как поясняет чё как: https://www.youtube.com/watch?v=Wz7RDh6CylI

**Reply by Aizistral** [reply]
![image](https://user-images.githubusercontent.com/47505981/141649373-cac0c95b-6fdf-4476-9854-b4404a491a28.png)
